Source/WebCore/platform/graphics/opentype/OpenTypeVerticalData.cpp

```diff
@@ -532,6 +534,7 @@ void OpenTypeVerticalData::getVerticalTranslationsForGlyphs(const Font* font, co
                 defaultVertOriginY = -m_defaultVertOriginY * sizePerUnit;
             outXYArray[1] = defaultVertOriginY;
             continue;
+            }
```

Comment:
This doesn't exactly match the chromium commit, it closes the bracket before `if (std::isnan(defaultVertOriginY))`.

Also, can you indent the new block?